apim-operator/pkg/controller/api/constants.go

```diff
@@ -84,7 +84,22 @@ const (
 	uploadFilesConst                    = "uploadFiles"
 	verifyHostnameConst                 = "verifyHostname"
 	hostnameConst                       = "hostname"
-	portConst                           = "port"
-	analyticsSecretConst                = "analyticsSecret"
-
+	portConst               = "port"
+	analyticsSecretConst    = "analyticsSecret"
+	authorizationUrl        ="https://example.com/oauth/authorize"
+	tokenUrl                ="https://example.com/oauth/token"
+	read                    ="read"
+	write                   ="write"
+	admin                   ="admin"
+	grantsReadAccess        ="Grants read access"
+	grantsWriteAccess       ="Grants write access"
+	grantsAccessToAdmin     ="Grants access to admin operations"
+	oauthSecurityType       ="oauth2"
```

Comment:
why did we hard code this scope values? I guess these are not mandatory in Swagger..

Reply:
added customize scope values. removed hard coded values.